src/index.html

```diff
@@ -13,10 +13,21 @@
     <title>Stop watch</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/index.css"
     />
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="stopwatch">
+      <div class="stopwatch__hand">
+        <div
+          class="stopwatch__hand stopwatch__hand--min stopwatch__hand--min--speed-up"
+        ></div>
```

Comment:
It's not a good practice to use multiple class names in a single element unless it's necessary. Here, you can simplify your classes and make your code cleaner by using SCSS nested rules or BEM methodology. For instance, instead of `stopwatch__hand stopwatch__hand--min stopwatch__hand--min--speed-up`, you can use `stopwatch__hand-min speed-up`.